src/components/App/App.css

```diff
@@ -0,0 +1,67 @@
+main {
+	font-family: 'Nunito', monospace;
```

Comment:
Can we standardize the indentation in here?

Reply:
In general, yes. I think in this case I removed in a subsequent commit.